pom.xml

```diff
@@ -1944,20 +1946,53 @@
       <build>
         <plugins>
           <plugin>
-            <groupId>org.jacoco</groupId>
-            <artifactId>jacoco-maven-plugin</artifactId>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-antrun-plugin</artifactId>
```

Comment:
I think that this deserves some documentation, at least in the form of an explanatory XML comment.

Reply:
@triceo what kind of documentation?  I would be fine with adding here some comments and show usage in [README.md](https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/master/README.md)

Reply:
Radek already added an XML comment, I consider that good enough.

Reply:
ok